functions.php

```diff
@@ -523,3 +526,100 @@ function twentytwenty_read_more_tag() {
 	add_filter( 'the_content_more_link', 'twentytwenty_read_more_tag' );
 
 }
+
+if ( ! function_exists( 'twentytwenty_customize_controls_enqueue_scripts' ) ) {
+	/**
+	 * Enqueues scripts for customizer controls & settings.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return void
+	 */
+	function twentytwenty_customize_controls_enqueue_scripts() {
+		$theme_version = wp_get_theme()->get( 'Version' );
+
+		// Add script for color calculations.
+		wp_enqueue_script( 'twentytwenty-color', get_template_directory_uri() . '/assets/js/color.js', [ 'wp-color-picker' ], $theme_version, false );
+
+		// Add script for controls.
+		wp_enqueue_script( 'twentytwenty-customize-controls', get_template_directory_uri() . '/assets/js/customize-controls.js', [ 'twentytwenty-color', 'customize-controls', 'underscore', 'jquery' ], $theme_version, false );
+		wp_localize_script( 'twentytwenty-customize-controls', 'backgroundColors', twentytwenty_get_customizer_color_vars() );
+	}
+
+	add_action( 'customize_controls_enqueue_scripts', 'twentytwenty_customize_controls_enqueue_scripts' );
+}
+
+if ( ! function_exists( 'twentytwenty_customize_preview_init' ) ) {
+	/**
+	 * Enqueue scripts for the customizer preview.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return void
+	 */
+	function twentytwenty_customize_preview_init() {
+		$theme_version = wp_get_theme()->get( 'Version' );
+
+		wp_enqueue_script( 'twentytwenty-customize-preview', get_theme_file_uri( '/assets/js/customize-preview.js' ), array( 'customize-preview', 'jquery' ), $theme_version, true );
+		wp_localize_script( 'twentytwenty-customize-preview', 'backgroundColors', twentytwenty_get_customizer_color_vars() );
+	}
+
+	add_action( 'customize_preview_init', 'twentytwenty_customize_preview_init' );
+}
+
+if ( ! function_exists( 'twentytwenty_get_color_for_area' ) ) {
+	/**
+	 * Get accessible color for an area.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param string $area The area we want to get the colors for.
+	 * @param string $context Can be 'text' or 'accent'.
+	 * @return string Returns a HEX color.
+	 */
+	function twentytwenty_get_color_for_area( $area = 'content', $context = 'text' ) {
+
+		// Get the value from the theme-mod.
+		$settings = get_theme_mod(
+			'accent_accessible_colors',
+			array(
+				'content'       => array(
+					'text'   => '#000000',
+					'accent' => '#cd2653',
+				),
+				'header-footer' => array(
+					'text'   => '#000000',
+					'accent' => '#cd2653',
+				),
+			)
+		);
+
+		// If we have a value return it.
+		if ( isset( $settings[ $area ] ) && isset( $settings[ $area ][ $context ] ) ) {
+			return $settings[ $area ][ $context ];
+		}
```

Comment:
Let's return false or an error if there isn't a value (for example if someone uses the wrong context).

Reply:
Sure, np. `null` (which is what gets returned now on error) does the same job, but I can see why some people would prefer false. Making the change.